browser/net/url_context.h

```diff
@@ -78,6 +78,7 @@ struct BraveRequestInfo {
   bool allow_http_upgradable_resource = false;
   bool allow_referrers = false;
   bool is_webtorrent_disabled = false;
+  bool resolve_ipfs_enabled = false;
```

Comment:
just a random thought - it seems that after all we should just save browser_context here

Reply:
That was actually my first thought, I prefer that way so ipfs details could be wrap inside ipfs's helper and not in the general url_context. I'm changing it to save browser_context directly here.